### ISSUE TITLE
TASK [Install TYPO3 project ...] fails

### ISSUE THREAD

**Issue by via-lars** [dachande/typo3-bootstrap#2]
Hello,

_TASK [Install TYPO3 project (this may take a while)]_ fails.

ssh ing into vagrant and running the command manually including debug output yields:
```
 - Installing typo3/cms-composer-installers (v2.2.2): Reading /home/vagrant/.cache/composer/files/typo3/cms-composer-installers/8d0e60b093da1707cb467eeee5e161ad07a30ab4.zip from cache
Loading from cache
Extracting archiveExecuting command (CWD): unzip -qq  '/var/www/typo3-website/vendor/typo3/cms-composer-installers/b131a52255655486fc8b61a6e9c75166' -d '/var/www/typo3-website/vendor/composer/324c77ca'
Plugin installation failed, rolling back
 - Removing typo3/cms-composer-installers (v2.2.2)
```

I am on Windows 10 (x64) and installed VirtualBox 6 and ist Extension Pack as well as Vagrant x64 and the Vagrant Plugins you mention prior to above attempt.

If i replace --prefer-dist by --prefer-source, Composer installs the TYPO3 Project but then the TYPO3 Setup Task errors out mentioning a syntax error in some project script.

Can you help me?

**Comment by via-lars** [comment]
I was succesful using box `ubuntu/bionic64` (in configuration.yml)

**Comment by dachande** [comment]
I'll have a look into this as this should not happen. But it dosn't look like a problem with the TYPO3 Bootstrap tool itself.

**Comment by dachande** [comment]
It looks like this problem is indeed related to the latest version of the bento/ubuntu-18.04 vagrant base box. Therefore I've updated the provisioning configuration to use the ubuntu/bionic64 base box as suggested. The problem has been fixed in the latest release 9.5.3
